src/Autofac.Extras.MvvmCross/AutoFacPropertyInjectionOptions.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Cirrious.CrossCore.IoC;
+
+namespace Autofac.Extras.MvvmCross
+{
+    /// <summary>
+    /// Adds some additional features to autofacpropertyinjection
```

Comment:
As for the comment before:
Calling Mvx.Resolve<T> on a T that has not been registered has to fail just as you said.
I only implicitly register types on Mvx.IoCConstruct<T>
In that case I do not care if T has been registered - it just should be automatically.
The basic MvvmCross IoC Container has exactly this behavior. :)